config.md

```diff
@@ -308,15 +308,23 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value are strings.
+Keys MUST be unique, and best practice is to namespace the keys.
+Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
+If there are no annotations then this property MAY either be absent or an empty map.
+Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
```

Comment:
And while I abbreviated the “Implementations that are reading/processing this configuration” wording in my suggestion above, I think we should probably just go with “[Runtimes][1]”.  Do you have other consumers in mind with your more general wording?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/glossary.md#runtime